Frameworks/CoreText/DWriteWrapper.mm

```diff
@@ -0,0 +1,395 @@
+//******************************************************************************
+//
+// Copyright (c) 2016 Microsoft Corporation. All rights reserved.
+//
+// This code is licensed under the MIT License (MIT).
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+//
+//******************************************************************************
+
+#include <COMIncludes.h>
+#import <wrl/implements.h>
+#include <COMIncludes_End.h>
+#import <mutex>
+#import <vector>
+#import <algorithm>
+#import <CoreFoundation/CFBase.h>
+#import "CoreTextInternal.h"
+#import "DWriteWrapper.h"
+
+using namespace std;
```

Comment:
I am going to leave it as is so I do not have to explicitly call std:: every place

---
In reply to: [77264108](https://github.com/Microsoft/WinObjC/pull/950#discussion_r77264108) [](ancestors = 77264108)